README.md

```diff
@@ -144,6 +145,37 @@ Disclaimer: currently parsing is done via simply splitting the strings so commas
 
 Not all the parameters can be used at both places. And there will be parameters which make sense only for one specific action only (or for a set of actions).
 
+### Parameters
```

Comment:
Let's move this before the section `Some complex examples how to specify a javaagent`.
and call it `Summary of Parameters`.

Reply:
Sure, will move `Parameters` and `Actions and supported parameters` before `Some complex examples how to specify a javaagent`